src/components/DonationWidget/components/DonationWizard.tsx

```diff
@@ -8,13 +8,18 @@ export const Container = styled.div`
 `;
 
 export const Input = styled.input`
-  ${tw`py-2 px-3 rounded-md border border-beige-500 focus:outline-none focus:border-blue w-full`}
+  ${tw`py-2 px-3 rounded-md bg-white border border-beige-500 focus:outline-none focus:border-blue w-full`}
 
+  & + #stripe-card-element,
```

Comment:
This type of things are not desired but have been introduced at 6ae1cb2df4e48bae084d3eb0688fe3abd22b2218 _(check description)_ as I had tech limitations to customise the input.